include/diaspora.php

```diff
@@ -456,7 +461,7 @@ public static function dispatch($importer, $msg) {
 	 *
 	 * @return bool Is the posting valid?
 	 */
-	private static function valid_posting($msg, &$fields) {
+	private static function valid_posting($msg) {
```

Comment:
please remove ` * @param object $fields SimpleXML object that contains the posting when it is valid` from the doxgen header